### ISSUE TITLE
use plenv instead of perlbrew in installation instructions?

### ISSUE THREAD

**Issue by nics** [LibreCat/Catmandu#226]

**Comment by phochste** [comment]
Done!

**Comment by netsensei** [comment]
I'm running into trouble using plenv instead of perlbrew.

After following the instructions, I get this error message upon executing `catmandu`:

````
Can't locate File/Find/Rule.pm in @INC (you may need to install the File::Find::Rule module) (@INC contains: /Users/netsensei/perl5/lib/perl5/darwin-2level /Users/netsensei/perl5/lib/perl5 /Users/netsensei/perl5/lib/perl5/darwin-2level /Users/netsensei/perl5/lib/perl5 /Users/netsensei/perl5/lib/perl5/darwin-2level /Users/netsensei/perl5/lib/perl5 /Users/netsensei/.plenv/versions/5.22.0/lib/perl5/site_perl/5.22.0/darwin-2level /Users/netsensei/.plenv/versions/5.22.0/lib/perl5/site_perl/5.22.0 /Users/netsensei/.plenv/versions/5.22.0/lib/perl5/5.22.0/darwin-2level /Users/netsensei/.plenv/versions/5.22.0/lib/perl5/5.22.0 .) at /Users/netsensei/perl5/lib/perl5/Data/Compare.pm line 16.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/lib/perl5/Data/Compare.pm line 16.
Compilation failed in require at /Users/netsensei/perl5/lib/perl5/Catmandu/Util.pm line 12.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/lib/perl5/Catmandu/Util.pm line 12.
Compilation failed in require at /Users/netsensei/perl5/lib/perl5/Catmandu/Error.pm line 156.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/lib/perl5/Catmandu/Error.pm line 156.
Compilation failed in require at /Users/netsensei/perl5/lib/perl5/Catmandu/Sane.pm line 13.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/lib/perl5/Catmandu/Sane.pm line 13.
Compilation failed in require at /Users/netsensei/perl5/lib/perl5/Catmandu.pm line 3.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/lib/perl5/Catmandu.pm line 3.
Compilation failed in require at /Users/netsensei/perl5/bin/catmandu line 3.
BEGIN failed--compilation aborted at /Users/netsensei/perl5/bin/catmandu line 3.
````

So, it complains about a missing module. When I install the module using cpanm and run catmandu again, it will complain about the next missing module. And so on.  Seems like cpanm doesn't install the dependencies needed.

Any ideas how to fix this?

**Comment by phochste** [comment]
Did you do a

   plenv rehash

when installing a new perl via plenv? You might be using a different cpan/cpanm then the one supplied by the perl.5.22

**Comment by netsensei** [comment]
Yup. I tried that multiple times without success.  

I didn't find much documentation about this issue so, I started trying out a few random things:

*  `plenv migrate-modules system 5.22.0` copied/installed extra modules, but not the deps from Catmandu like Class::Load.
* running `plenv local 5.22.0`  and then force reinstalling Catmandu did seem to push it to download a few extra dependencies.

By that time, I already installed most of the missing dependencies manually. I got catmandu running, but I assume there are still missing dependencies lurking below the water when I start using it.
